money-monkey-web/app/ui/CategoryList.tsx

```
@@ -1,28 +1,39 @@
   return (
     <div className="w-full flex flex-col gap-1">
-      {sums.map((sum) => (
-        <div
-          key={sum.category_id}
-          className="flex justify-between p-1 hover:bg-blue-100"
-        >
-          <div className="flex items-center gap-2">
+      {sums
+        .filter((sum) => categories.has(sum.category_id))
+        .map((sum) => {
+          const cat = categories.get(sum.category_id)!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)